labelme/app.py

```
@@ -6,7 +6,12 @@
 import os
 import os.path as osp
 import re
+from typing import OrderedDict
```

Ruff (0.8.2)

`typing.OrderedDict` imported but unused

Remove unused import: `typing.OrderedDict`

(F401)